### PR TITLE
docs: restructure files to add homepage

### PR DESCRIPTION
**Pull request by robel-yemane**
adds index.md to create a homepage that contains intro to Argus.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const config = {
   url: `https://${organizationName}.github.io`,
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: `/${projectName}/`,
+  baseUrl: `/`,
```
Comment (onselakin) [inline docusaurus.config.js:19]: Why the change? If you tinker with it then the published site stops working :)

**Review by onselakin — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (robel-yemane) [reply docusaurus.config.js:19]: my thinking here is that since `${projectName}/` points to `docs` the homepage would be at https://compliance-framework.github.io/docs/ with this change however, the main page would be the root i.e. https://compliance-framework.github.io/, at least that corresponds to my local deployment with yarn
Argus_Cloud_Compliance_Framework___Argus_-_Cloud_Compliance_Framework

Comment (onselakin) [reply docusaurus.config.js:19]: Unfortunately, it doesn't work like that, at least not to my knowledge. It works on localhost but fails on Github. It'd be great if you could make it work though.

Comment (robel-yemane) [reply docusaurus.config.js:19]: oh! Ok let me try a couple of things. I think you need to remove the change request in the PR for me to merge it
